content/docs/instrumenting/clientlibs.md

```diff
@@ -21,6 +21,7 @@ HTTP endpoint on your application’s instance:
 Unofficial third-party client libraries:
 
 * [Bash](https://github.com/aecolley/client_bash)
+* [Clojure](https://github.com/soundcloud/prometheus-clj) (wraps the Java client library)
```

Comment:
Also I'm strongly against wrappers of client libraries, I'd want to give this a thorough review to ensure it follows the client guidelines and doesn't distort the java client before even mentioning it in our docs